js/src/index.js

```diff
@@ -29,6 +29,9 @@ import qs from 'querystring';
 
 import SecureApi from './secureApi';
 import ContractInstances from './contracts';
+import { initLocales } from './i18n';
+
+initLocales('en');
```

Comment:
I find the naming `initLocales` confusing. It sounds as if you could pass one or more locales to be initialised, in this case only English. However, underneath, *all* locales will be initialised, with English being the chosen one.